doc/nrf/gs_programming.rst

```diff
@@ -186,7 +186,11 @@ Complete the following steps to build |NCS| projects on the command line after c
 
          west flash
 
-      To fully erase the board before programming the new sample or application, use the command:
+      This command clears only the flash memory pages that are overwritten with the new sample or application.
```

Comment:
Sample seems redundant here.

```suggestion
      This command clears only the flash memory pages that are overwritten with the new application.

```

Reply:
That's because we try to make a distinction between sample and application in the NCS - but here it doesn't really make sense. I removed some of the "sample or application" that are really not needed. :)